src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -394,8 +394,10 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               OldFanVO(NumOldFanVO)%coeff5 = TRIM(IDFRecords(Num)%Numbers(12))             !- Coefficient5 x**4
               OldFanVO(NumOldFanVO)%inletAirNodeName = TRIM(IDFRecords(Num)%Alphas(4))
               OldFanVO(NumOldFanVO)%outletAirNodeName = TRIM(IDFRecords(Num)%Alphas(5))
-              IF (lEN(IDFRecords(Num)%Alphas) .eq. 6) THEN
+              IF (SIZE(IDFRecords(Num)%Alphas) .eq. 6) THEN
```

Comment:
Ahh good fix.  Yeah, LEN on an CHARACTER ARRAY returns the length of each string inside the array, not the size of the array itself.  Subtle.